src/adhocracy_mercator/adhocracy_mercator/scripts/export_proposals.py

```diff
@@ -1,20 +1,18 @@
-"""Export scripts for adhocracy_mercator.
+"""Export proposal.
```

Comment:
@slomo need to update the cron script on the production server once this branch is merged